configure

```diff
@@ -1337,7 +1342,8 @@ do
     make_dir $t/rt/compiler-rt
     for i in                                          \
       isaac sync test \
-      arch/i386 arch/x86_64 arch/arm arch/aarch64 arch/mips arch/powerpc
+      arch/i386 arch/x86_64 arch/arm arch/aarch64 arch/mips arch/powerpc \
+      arch/powerpc64 arch/powerpc64le
```

Comment:
It should be fine to not update this, these are largely just all a bunch of old directories that aren't used any more